contracts/LifMarketMaker.sol

```diff
@@ -97,22 +106,64 @@ contract LifMarketMaker is Ownable {
     ];
 
     uint256 accumDistribution = 0;
-    uint256 periodStartBlock = startBlock;
+    uint256 deltaDistribution = 0;
+    uint256 startBlockPeriod = startBlock;
 
     for (uint8 i = 0; i < totalPeriods; i++) {
-      uint256 deltaDistribution;
+
+      require(marketMakerPeriods.length <= i);
```

Comment:
what are we validating here? seems like we can just `require(marketMakerPeriods.length == 0)` before the for loop